modules/administration-guide/pages/installing-che-on-codeready-containers.adoc

```diff
@@ -10,10 +10,73 @@
 
 :context: installing-{prod-id-short}-on-codeready-containers
 
-This chapter describes the creation of a single-node OpenShift cluster for deploying a {prod-short} instance using CodeReady Containers. To install {prod-short}, use the`{prod-cli}` command-line tool or the OpenShift web console GUI.
+You can create a single-node OpenShift cluster for deploying a {prod-short} instance using CodeReady Containers.
 
-include::partial$proc_using-chectl-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
+.Prerequisites
 
-include::partial$proc_using-openshift-web-console-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
+* `{prod-cli}`. See: xref:installing-the-chectl-management-tool.adoc[].
 
-:context: {parent-context-of-installing-che-on-codeready-containers}
+* A running instance of CodeReady Containers. See link:https://console.redhat.com/openshift/create/local[Installing CodeReady Containers].```
```

Comment:
CRC is now called RH OpenShift Local.

Reply:
Indeed, but Eva is already working on it.